policy/modules/services/zfs.te

```diff
@@ -129,6 +129,8 @@ userdom_use_user_terminals(zfs_t)
 zfs_rw_zpool_cache(zfs_t)
 
 optional_policy(`
+	fsadm_runtime_filetrans(zfs_t, dir, "blkid")
+
```

Comment:
I would put this in it's own 'optional_policy' block so it isn't reliant on rpc being enabled also.

Reply:
Done.